sql/synth_recipes.sql

```diff
@@ -3597,25 +3597,25 @@ INSERT INTO `synth_recipes` VALUES (59928,1,0,0,0,0,0,0,7,0,0,4100,4242,13442,0,
--- INSERT INTO `synth_recipes` VALUES (59939,1,0,0,0,0,0,0,255,0,0,4100,4242,14008,0,0,0,0,0,0,0,817,1193,850,850,3,1,3,3,'Cpc. Gauntlets (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59940,1,0,0,0,0,0,0,255,0,0,4100,4242,17849,0,0,0,0,0,0,0,2157,2171,2171,2171,1,1,1,1,'Hellish Bugle (desynth)');
+INSERT INTO `synth_recipes` VALUES (59933,1,0,0,0,0,0,0,72,0,0,4100,4242,12878,0,0,0,0,0,0,0,819,506,887,887,2,1,1,1,'Coral Subligar (desynth)');
+INSERT INTO `synth_recipes` VALUES (59934,1,0,0,0,0,0,0,44,0,0,4100,4242,13713,0,0,0,0,0,0,0,864,864,864,864,1,1,1,1,'Carapace Mittens (desynth)');  -- 2021.11.01
+INSERT INTO `synth_recipes` VALUES (59935,1,0,0,0,0,0,0,39,0,0,4100,4242,17026,0,0,0,0,0,0,0,817,880,880,893,3,5,6,1,'Bone Cudgel (desynth)');  -- 2021.11.01 XI and allakhzam have different recipes.  Went with https://ffxiclopedia.fandom.com/wiki/Bone_Cudgel
+INSERT INTO `synth_recipes` VALUES (59936,1,0,0,0,0,0,0,77,0,0,4100,4242,13850,0,0,0,0,0,0,0,887,792,793,793,1,1,1,1,'Merman\'s Hairpin (desynth)');  -- 2021.11.01 https://ffxiclopedia.fandom.com/wiki/Merman's_Hairpin
+INSERT INTO `synth_recipes` VALUES (59937,1,0,0,0,0,0,0,36,0,0,4100,4242,12583,0,0,0,0,0,0,0,852,889,852,852,1,1,2,3,'Beetle Harness (desynth)');  -- 2021.11.01 nothing matches. XI makes no sense b/c of materials needed to make.  Used https://ffxi.allakhazam.com/db/item.html?fitem=536#Desynthesis
+INSERT INTO `synth_recipes` VALUES (59938,1,0,0,0,0,0,0,97,0,0,4100,4242,13846,0,0,0,0,0,0,0,850,1230,1473,1473,1,6,1,1,'Scorpion Helm (desynth)');  -- 2021.11.01
+INSERT INTO `synth_recipes` VALUES (59939,1,0,0,0,0,0,0,92,0,0,4100,4242,14008,0,0,0,0,0,0,0,817,1193,850,850,3,1,1,3,'Cpc. Gauntlets (desynth)');  -- 2021.11.01 allakhazam states synth rate is 3,1,1,3 XI shows 3,3,1,3. Nothing shown on others. Chose 3,1,1,3 b/c there are not 3 Crab Shells in the synths to make.
+INSERT INTO `synth_recipes` VALUES (59940,1,0,0,0,0,0,0,85,0,0,4100,4242,17849,0,0,0,0,0,0,0,2157,2157,2171,2171,1,1,1,1,'Hellish Bugle (desynth)');  -- 2021.11.01
 INSERT INTO `synth_recipes` VALUES (59941,1,0,0,0,0,0,0,80,0,0,4100,4242,15211,0,0,0,0,0,0,0,887,887,792,793,1,1,1,1,'Reraise Hairpin (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59942,1,0,0,0,0,0,0,255,0,0,4100,4242,13108,0,0,0,0,0,0,0,850,887,887,887,1,1,1,1,'Coral Gorget (desynth)');
+INSERT INTO `synth_recipes` VALUES (59942,1,0,0,0,0,0,0,79,0,0,4100,4242,13108,0,0,0,0,0,0,0,850,850,887,887,1,1,1,1,'Coral Gorget (desynth)');  -- 2021.11.01
 INSERT INTO `synth_recipes` VALUES (59943,1,0,0,0,0,0,0,70,0,0,4100,4242,13464,0,0,0,0,0,0,0,864,864,902,902,1,1,1,1,'Demon\'s Ring (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59944,1,0,0,0,0,0,0,255,0,0,4100,4242,12436,0,0,0,0,0,0,0,850,866,866,866,1,2,2,2,'Dragon Mask (desynth)');
+INSERT INTO `synth_recipes` VALUES (59944,1,0,0,0,0,0,0,87,0,0,4100,4242,12436,0,0,0,0,0,0,0,850,850,866,866,1,1,1,2,'Dragon Mask (desynth)');  -- 2021.11.01 XI and allakhazam
```

Comment:
Can we expand out "XI"? I assume you mean ffxiclopedia, but only because when I spoke Raist that's what he meant using it. We have a ton of things in the ffxi community that are "xi".

Reply:
same for other instances of "XI"

Reply:
Yeah... I noticed a dozen or so from my previous entries that need clarification/cleanup as well.  Plan to stitch the edits in with future submissions.

Reply:
Yeah, I can do that.